x-pack/plugins/security_solution/public/common/components/links_to_docs/links_translations.ts

```diff
@@ -33,3 +33,11 @@ export const DETECTIONS_REQUIREMENTS_LINK_TEXT = i18n.translate(
     defaultMessage: 'Detections prerequisites and requirements',
   }
 );
+
+export const ML_JOB_COMPATIBILITY_LINK_PATH = 'alerts-ui-monitor.html#ml-job-compatibility';
```

Comment:
Just commenting to show the full URL I saw in testing:


https://www.elastic.co/guide/en/security/master/alerts-ui-monitor.html#ml-job-compatibility